docs/release-notes/2.20.0.md

```diff
@@ -0,0 +1,60 @@
+- PostgreSQL monitoring with the `pg_stat_monitor` plugin enabled exposes some new metrics, such as Plan Times, WAL Fpi/Bytes/Records.
+- For users who use PMM through the AWS Marketplace, AWS RDS service discovery will be executed without AWS credentials and tuning [IAM roles].
+
+[Easy-install script]: https://www.percona.com/doc/percona-monitoring-and-management/2.x/setting-up/server/easy-install.html
+[grafana.com]: https://grafana.com/docs/grafana/latest/whatsnew/whats-new-in-v7-5
+[IAM roles]: https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles.html
+
+## New Features
+
+- [PMM-8157](https://jira.percona.com/browse/PMM-8157): Backup Management: Show scheduled backups
+- [PMM-8155](https://jira.percona.com/browse/PMM-8155): Backup Management: Scheduling of backups
+- [PMM-7010](https://jira.percona.com/browse/PMM-7010): Option to unregister current node (`pmm-admin unregister`)
+
+## Improvements
+
+- [PMM-7649](https://jira.percona.com/browse/PMM-7649): Show "Insufficient access permissions" for non admin users
+- [PMM-7552](https://jira.percona.com/browse/PMM-7552): Add labels to PMM Server Docker image (Thanks to Nicolas for reporting this issue)
+- [PMM-8266](https://jira.percona.com/browse/PMM-8266): PMM Server Docker image: Decommission and remove Prometheus binaries and configuration
+- [PMM-8040](https://jira.percona.com/browse/PMM-8040): PMM Server Docker image: Remove yum cache to reduce size of image
+- [PMM-8000](https://jira.percona.com/browse/PMM-8000): Expose new numbered metrics available in `pg_stat_monitor` 0.9
+- [PMM-7809](https://jira.percona.com/browse/PMM-7809): Grafana upgrade from 7.3.7 to 7.5.7 – [Read more at grafana.com](https://grafana.com/docs/grafana/latest/whatsnew/whats-new-in-v7-5)
+- [PMM-7722](https://jira.percona.com/browse/PMM-7722): DBaaS: Announce new supported version of operator -- Shows that a new version of the operator is available.
+
+## Bugs Fixed
+
+- [PMM-6877](https://jira.percona.com/browse/PMM-6877): `mongodb_exporter` doesn't recognize being on a mongos host and fills the syslog with replSetGetStatus errors (Thanks to Clyde Shillingford for reporting this issue)
+- [PMM-7690](https://jira.percona.com/browse/PMM-7690): AWS discovery and monitoring based on IAM roles
+- [PMM-8411](https://jira.percona.com/browse/PMM-8411): Discovery starts immediately on Add Instance page
+- [PMM-8405](https://jira.percona.com/browse/PMM-8405): MySQL/PostgreSQL/MongoDB Instance Summary and Node Summary dashboards: PT Summary Panel broken after upgrade
+- [PMM-7627](https://jira.percona.com/browse/PMM-7627): Consistent PMM Server log management -- Adds consistency to the log management of `nginx`, `postgresql` and `clickhouse-server`, which is now delegated to `supervisord`. Removes the `logrotate` daemon from the image.
+- [PMM-8299](https://jira.percona.com/browse/PMM-8299): Backup Management: Remove storage location - showing success notification when there's an error
```

Comment:
Backup Management: When removing a storage location, PMM presents misleading information to the user in the form of two notification messages for error and success.

Reply:
updated the subject

Reply:
I don't see any changes either here or in the originating Jira issue (https://jira.percona.com/browse/PMM-8299).

Reply:
JIRA issue has been updated to reflect this change.